test/Microsoft.ML.TestFramework/DataPipe/TestDataPipe.cs

```diff
@@ -145,4 +145,24 @@ public void TestLdaTransformEmptyDocumentException()
             Assert.True(false, "The LDA transform does not throw expected error on empty documents.");
         }
     }
+
+    public sealed partial class TestDataPipe : TestDataPipeBase
+    {
+
+        [Fact]
+        public void TestParquetPrimitiveDataTypes()
+        {
+            string pathData = GetDataPath(@"..\data\Parquet", "alltypes.parquet");
```

Comment:
`GetDataPath` is already at the `test\data` folder.  You shouldn't need `..\data\` since that just gets you back into the same folder.  You should just have `GetDataPath("Parquet", "alltypes.parquet");`